lib/src/doc/lives.rs

```diff
@@ -41,6 +42,13 @@ impl<'a> Document<'a> {
 						continue;
 					}
 				}
+				// Check authorization
+				trace!("Checking live query auth: {:?}", lv);
+				let lq_options = Options::new_with_perms(opt, true)
+					.with_auth(Arc::from(lv.auth.clone().ok_or(Error::UnknownAuth)?));
+				if self.allow(ctx, &lq_options, txn, &lq).await.is_err() {
+					continue;
+				}
```

Comment:
in relation to the comment made to the `ws_integration.rs` file, I think it's important we have unit tests making sure the auth is enforced properly. Testing it as part of the integration tests, given the nature of the feature, could give us a false sense of security.

Reply:
Agreed to follow up, SUR-261